src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/DiagnosticsClient.cs

```diff
@@ -156,59 +156,103 @@ public void AttachProfiler(TimeSpan attachTimeout, Guid profilerGuid, string pro
             // runtime timeout or respect attachTimeout as one total duration.
         }
 
-        internal void ResumeRuntime()
+        /// <summary>
+        /// Set a profiler as the startup profiler. It is only valid to issue this command
+        /// while the runtime is paused in the "reverse server" mode.
+        /// </summary>
+        /// <param name="profilerGuid">Guid for the profiler to be attached</param>
+        /// <param name="profilerPath">Path to the profiler to be attached</param>
+        public void SetStartupProfiler(Guid profilerGuid, string profilerPath)
```

Comment:
We don't currently have a notification mechanism for informing a client that the target is in suspension. Are you anticipating the generic `Profiler startup failed ...` message to handle that response?

This does make me think we should have an error message for "not available during suspension"

Reply:
It seems to currently return DS_IPC_E_INVALIDARG when suspended.